spy_terminal.py

```diff
@@ -27,76 +30,94 @@ def main():
     while x != 'boom':
         if welcome():
             password = set_up()
-            scrambled_message = new_message(codex,count)
+            scrambled_message = new_message(codex, count)
             count = 1
         else:
-            retrieve_message(password, scrambled_message,codex,count)
+            retrieve_message(password, scrambled_message, codex, count)
             scrambled_message = None
             count = 0
         x = input("To self-destruct type 'boom'. Otherwise, hit enter. ")
-    self_destruct(5,'KABOOM', "Yes")
+    self_destruct(5, 'KABOOM', "Yes")
+
 
 '''---------------------SUPPORTING FUNCTIONS ------------------------'''
 
 '''Welcomes the user and directs them to either write a new message or retrieve an existing message.'''
+
+
 def welcome():
     today = date.today()
     print('Welcome Agent. The current date is:', today)
-    desire = input("Enter 'L' to leave a new message. Enter 'R' to retrieve message: ")
+    desire = input(
+        "Enter 'L' to leave a new message. Enter 'R' to retrieve message: ")
     while desire != 'L' and desire != 'R':
-        desire = input("Enter 'L' or leave a new message. Enter 'R' to retrieve message: ")
+        desire = input(
+            "Enter 'L' or leave a new message. Enter 'R' to retrieve message: ")
     if desire == 'L':
         return True
     else:
         return False
 
+
 '''Uses count to check if a message already exists. If so, warns this will be overridden. 
     Takes message and puts it through codex to scramble and then override it.'''
-def new_message(codex,count):
+
+
+def new_message(codex, count):
     if count > 0:
         print("Warning! You already have a message logged. "
               "Your new message will replace it. ")
         time.sleep(1)
         decision = input("Continue? Y/N: ")
         if decision == 'Y':
-            entry = input("Please give me your message. Note: Do not use numbers: ")
+            entry = input(
+                "Please give me your message. Note: Do not use numbers: ")
             scrambled_message = scramble(entry, codex)
             return scrambled_message
     if count < 1:
-        entry = input("Please give me your message. Note: Do not use numbers: ")
+        entry = input(
+            "Please give me your message. Note: Do not use numbers: ")
         scrambled_message = scramble(entry, codex)
         return scrambled_message
 
+
 '''This function takes in an existing password, a coded message and the codex. 
     It determines whether the password is correct and if so unscrambles the message and prints it for the user.
     A count lets user know if no messages have been left. 
     It also erases the message automatically'''
```

Comment:
Docstrings are really helpful, and it's wonderful you included them! They're best placed directly after the definition name of a function and indented so that they're clearly an attribute of the function itself, as you did with `main()`. When I write docstrings, I tend to include a brief description of the function's purpose and also define what its inputs and outputs (and their types) are.
```suggestion
    """
    Prints a given message if a valid password is provided
    
    :param password str
    :param coded_message str
    :param codex str
    :param count int
    """
```